build.gradle

```diff
@@ -19,8 +19,8 @@ buildscript {
 
 allprojects {
     repositories {
-        jcenter()
         google()
```

Comment:
@letelete Undo this change

Reply:
Pull request #909 covers this change, and a fix for multidex